test/src/audits/manifest/short-name.js

```diff
@@ -34,15 +34,26 @@ describe('Manifest: short_name audit', () => {
 
   // Need to disable camelcase check for dealing with short_name.
   /* eslint-disable camelcase */
-  it('fails when a manifest contains no short_name', () => {
+  it('fails when a manifest contains no short_name and no name', () => {
```

Comment:
oh, it's up in `short-name-length.js`. I meant to replace this test here in this file